src/pages/VoterDetail/History/index.tsx

```diff
@@ -0,0 +1,85 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import { Typography } from '@mui/material';
+import { Pagination, Spinner } from 'components';
+import { useTranslation } from 'translation';
+import { IVoterHistory } from 'types';
+import VoterProposal from './VoterProposal';
+import { useStyles } from './styles';
+
+interface IHistoryProps {
+  className?: string;
+  voterHistory: IVoterHistory[] | undefined;
+  setCurrentPage: (page: number) => void;
+  total: number;
+  limit: number;
+  isFetching: boolean;
+}
+
+export const History: React.FC<IHistoryProps> = ({
+  className,
+  voterHistory,
+  setCurrentPage,
+  total,
+  limit,
+  isFetching,
+}) => {
+  const styles = useStyles();
+  const { t } = useTranslation();
+  return (
+    <div className={className}>
+      <Typography variant="h4">{t('voterDetail.votingHistory')}</Typography>
+      {voterHistory && !isFetching ? (
+        voterHistory.map(
+          ({
+            proposal: {
+              id,
+              description,
+              state,
+              endDate,
+              forVotesWei,
+              abstainedVotesWei,
+              againstVotesWei,
+              createdDate,
+              queuedDate,
+              cancelDate,
+              executedDate,
+            },
+            support,
+          }) => (
+            <VoterProposal
+              key={id}
+              proposalNumber={id}
+              proposalTitle={description.title}
+              proposalState={state}
+              forVotesWei={forVotesWei}
+              againstVotesWei={againstVotesWei}
+              abstainedVotesWei={abstainedVotesWei}
+              userVoteStatus={support}
+              createdDate={createdDate}
+              queuedDate={queuedDate}
+              cancelDate={cancelDate}
+              executedDate={executedDate}
+              endDate={endDate}
+            />
+          ),
+        )
+      ) : (
+        <Spinner />
+      )}
+      {total ? (
+        <Pagination
+          css={styles.pagination}
+          itemsCount={total}
+          onChange={(nextIndex: number) => {
+            setCurrentPage(nextIndex);
+            window.scrollTo(0, 0);
+          }}
+          itemsPerPageCount={limit}
+        />
+      ) : null}
```

Comment:
```suggestion
      {total > 0 && (
        <Pagination
          css={styles.pagination}
          itemsCount={total}
          onChange={(nextIndex: number) => {
            setCurrentPage(nextIndex);
            window.scrollTo(0, 0);
          }}
          itemsPerPageCount={limit}
        />
      )}
```

Reply:
This is actually handled internally by Pagination, the check exists because `itemsCount` can't be undefined